core/src/main/java/com/radixdlt/consensus/LedgerHeader.java

```diff
@@ -108,6 +109,10 @@ public final class LedgerHeader {
   @DsonOutput(Output.ALL)
   private final AccumulatorState accumulatorState;
 
+  @JsonProperty("state_hash")
+  @DsonOutput(value = Output.HASH, include = false)
```

Comment:
I think we want to include it when we save/load and when we send it over the API, but just not include it for hashing, right? so EG:
```
  @DsonOutput(value = {Output.API, Output.WIRE, Output.PERSIST})
```

Reply:
> I think we want to include it when we save/load and when we send it over the API, but just not include it for hashing, right?

Yes. And `@DsonOutput(value = Output.HASH, include = false)` means exactly "use it for every output except HASH". That's how I interpret the `include` javadoc and in the impl:
https://github.com/radixdlt/babylon-node/blob/93df7f8a81f480371b070e1c33399aef0dcdd43a/common/src/main/java/com/radixdlt/serialization/DsonOutput.java#L171-L172
(and the runtime behaviour seems to match this expectation)

Reply:
Ahh cool! Nice - didn't know that 👍